cassovary-benchmarks/src/main/scala/com/twitter/cassovary/PageRankBenchmark.scala

```diff
@@ -24,4 +24,4 @@ class PageRankBenchmark(graph: DirectedGraph[Node],
   def operation() {
     PageRank(graph, pageRankParams)
   }
-}
\ No newline at end of file
+}
```

Comment:
After trying to figure out how to run benchmarks for a while, I decided to just ask.  How do I run the benchmarks?  Thanks!

I will probably update the README with instructions on how to run them, unless of course I'm just being an idiot.